### PR TITLE
Feature/set online offline

### PR DESCRIPTION
**Pull request by mkryva**

### CODE REVIEW

**Comment by akuzminsky** [comment]
When I set offline a writer it goes OFFLINE_HARD, but next `galera register` sets it back to ONLINE

```
[vagrant@proxysql proxysql_tools]$ proxysql-tool galera server set_offline 192.168.90.2
[vagrant@proxysql proxysql_tools]$ proxysql-tool galera server status
2017-08-02 17:42:10,539: INFO: server.server_status():31: Writers:
+--------------+--------------+------+--------------+--------+-------------+-----------------+---------------------+---------+----------------+----------------------------------------------------+
| hostgroup_id | hostname     | port |       status | weight | compression | max_connections | max_replication_lag | use_ssl | max_latency_ms | comment                                            |
+--------------+--------------+------+--------------+--------+-------------+-----------------+---------------------+---------+----------------+----------------------------------------------------+
|           10 | 192.168.90.2 | 3306 | OFFLINE_HARD |      1 |           0 |           10000 |                   0 |   False |              0 | {"admin_status": "OFFLINE_HARD", "role": "Writer"} |
+--------------+--------------+------+--------------+--------+-------------+-----------------+---------------------+---------+----------------+----------------------------------------------------+
2017-08-02 17:42:10,544: INFO: server.server_status():31: Readers:
+--------------+--------------+------+--------+--------+-------------+-----------------+---------------------+---------+----------------+----------------------------------------------+
| hostgroup_id | hostname     | port | status | weight | compression | max_connections | max_replication_lag | use_ssl | max_latency_ms | comment                                      |
+--------------+--------------+------+--------+--------+-------------+-----------------+---------------------+---------+----------------+----------------------------------------------+
|           11 | 192.168.90.3 | 3306 | ONLINE |      1 |           0 |           10000 |                   0 |   False |              0 | {"admin_status": "ONLINE", "role": "Reader"} |
|           11 | 192.168.90.4 | 3306 | ONLINE |      1 |           0 |           10000 |                   0 |   False |              0 | {"admin_status": "ONLINE", "role": "Reader"} |
+--------------+--------------+------+--------+--------+-------------+-----------------+---------------------+---------+----------------+----------------------------------------------+
[vagrant@proxysql proxysql_tools]$ proxysql-tool galera register
2017-08-02 17:42:17,806: WARNING: load_balancing_mode.register_writer():114: No writer backends were registered. Will try to add previously ignored backends
2017-08-02 17:42:17,822: INFO: load_balancing_mode.register_synced_backends():315: Added backend hostgroup_id=10, hostname=192.168.90.2, port=3306, status=ONLINE, weight=1, compression=0, max_connections=10000, max_replication_lag=0, use_ssl=False, max_latency_ms=0, comment=None to hostgroup 10
[vagrant@proxysql proxysql_tools]$ proxysql-tool galera server status
2017-08-02 17:42:34,989: INFO: server.server_status():31: Writers:
+--------------+--------------+------+--------+--------+-------------+-----------------+---------------------+---------+----------------+----------------------------------------------+
| hostgroup_id | hostname     | port | status | weight | compression | max_connections | max_replication_lag | use_ssl | max_latency_ms | comment                                      |
+--------------+--------------+------+--------+--------+-------------+-----------------+---------------------+---------+----------------+----------------------------------------------+
|           10 | 192.168.90.2 | 3306 | ONLINE |      1 |           0 |           10000 |                   0 |   False |              0 | {"admin_status": "ONLINE", "role": "Writer"} |
+--------------+--------------+------+--------+--------+-------------+-----------------+---------------------+---------+----------------+----------------------------------------------+
2017-08-02 17:42:34,995: INFO: server.server_status():31: Readers:
+--------------+--------------+------+--------+--------+-------------+-----------------+---------------------+---------+----------------+----------------------------------------------+
| hostgroup_id | hostname     | port | status | weight | compression | max_connections | max_replication_lag | use_ssl | max_latency_ms | comment                                      |
+--------------+--------------+------+--------+--------+-------------+-----------------+---------------------+---------+----------------+----------------------------------------------+
|           11 | 192.168.90.3 | 3306 | ONLINE |      1 |           0 |           10000 |                   0 |   False |              0 | {"admin_status": "ONLINE", "role": "Reader"} |
|           11 | 192.168.90.4 | 3306 | ONLINE |      1 |           0 |           10000 |                   0 |   False |              0 | {"admin_status": "ONLINE", "role": "Reader"} |
+--------------+--------------+------+--------+--------+-------------+-----------------+---------------------+---------+----------------+----------------------------------------------+
```

**Comment by mkryva** [comment]
Done

File: proxysql_tools/cli.py [hunk 1]
```diff
@@ -199,6 +199,24 @@ def set_sync(cfg, ip_address, port):
         LOG.error(err)
 
 
+@server.command()
+@click.argument('ip_address', required=True)
+@click.argument('port', required=False, type=int, default=3306)
+@PASS_CFG
+def set_online(cfg, ip_address, port):
+    """Set server backend status online"""
+    server_set_admin_status(cfg, ip_address, port, BackendStatus.online)
```
Comment (akuzminsky) [inline proxysql_tools/cli.py]: Passing cfg down is a bad design practice. Let's discuss offline how we can refactor it

**Comment by akuzminsky** [comment]
This pr is reimplemented in #50

**Pull request closed without merging**